src/HandlerParser/parser.go

```diff
@@ -0,0 +1,135 @@
+package handlerparser
```

Comment:
It would be better to have a `parser` package with several parts inside it (endpoint, info, handler...)

Reply:
true but we re probably gonna do that when we redo all the parser to read the file only 1 time and handle folders

So can we let this through and correct it in the current sprint (sorry i dunno wats the sprint number)